lib/maths/CQDigest.cc

```diff
@@ -103,7 +109,7 @@ void CQDigest::add(uint32_t value, uint64_t n) {
         this->compress(compress);
     }
 
-    //this->checkInvariants();
+    this->checkInvariants();
```

Comment:
I don't think it's necessary to check here, as the input to this method didn't come from user input.

The check was previously commented out due to the performance impact.  Uncommenting it doesn't just do the check once per run like it would after restoring from persisted state, but potentially millions of times.